packages/contracts/tests/Poll.test.ts

```diff
@@ -92,6 +92,10 @@ describe("Poll", () => {
       expect(p.toString()).to.eq(pollId.toString());
       // publish the NOTHING_UP_MY_SLEEVE message
       const messageData = [NOTHING_UP_MY_SLEEVE];
+      expect(
+        NOTHING_UP_MY_SLEEVE === BigInt("8370432830353022751713833565135785980866757267633941821328460903436894336785"),
+      );
```

Comment:
I'm not sure if I makes sense to check the constant value in tests.
I think the idea is to check this value is inside the tree. Checking the constant value doesn't give any coverage.